GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -3152,6 +3153,23 @@ private void Message_Enter(object sender, EventArgs e)
                 StagedSelectionChanged(null, null);
             }
         }
+
+        internal TestAccessor GetTestAccessor()
+            => new TestAccessor(this);
```

Comment:
:memo: This is a common pattern we used at a former company to avoid production code "leaking" private objects. We found that consistent naming made it virtually impossible for any use of `GetTestAccessor` in production code to make it through code review.